### ISSUE TITLE
error while running plantuml

### ISSUE THREAD

**Issue by danieleades** [useblocks/sphinxcontrib-needs#151]
I'm unable to generate a `needflow`

```
WARNING: error while running plantuml

b'ERROR\n21\nSyntax Error?\nSome diagram description contains errors\n'
```

it works perfectly for my `requirements`, but i have another custom need called a `scenario` with the directive '`scn`'. Any needflow which is filtered to contain scenarios throws this syntax error. Even if these are completely empty.

Is there a way I can see the raw plantuml that's generated?

**Comment by danwos** [comment]
Yes, please use ``debug`` option. Docs: https://sphinxcontrib-needs.readthedocs.io/en/latest/directives/needflow.html#needflow-debug

**Comment by danieleades** [comment]
here is the debug output with a minimal reproduction-

```
@startuml

' Nodes definition 

node "<size:12>Scenario</size>\n**User**\n<size:10>SCN_001</size>" as SCN_001 [[../scenarios/SCN_001.html#SCN_001]] # 

' Connection definition 


@enduml
```

**Comment by danieleades** [comment]
the empty `#` on the end is the colour, isn't it? I've got an empty string for the colour in my custom 'need', so i'm guessing that's what's causing problems.

closing

**Comment by danwos** [comment]
Yes, ``#`` indicates the usage of a color.
Sphinx-Needs maybe should check, if the color was set correctly. 
I'll open the issue again and change the title, so I have a reminder for a small fix for the next release.
